README.md

```diff
@@ -25,14 +23,10 @@ The service (back-end):
 The UI (front-end):
   - is defined in main.py
   - allows stopping/restarting the backend.
-  - displays the last received messages from the backend in a RecycleView
-  - allow to sent a '/ping' message to the backend, which will trigger a new message.
+  - sends the last received messages as a notification
```

Comment:
ah, i though you would add the feature, not replace the existing one, the example surely can do both.